fusion-cli/build/load-fusionrc.js

```diff
@@ -16,10 +16,16 @@ const chalk = require('chalk');
 let loggedNotice = false;
 
 /*::
+
+type CompileResult = {
+  bundle: 'both' | 'client',
+  transpile: 'spec' | 'all' | 'none',
```

Comment:
I personally think `transpile` is a misleading term because this is so strongly associated with ES2017 -> ES5, but this is also about macros/loaders.

```suggestion
  transform: 'all' | 'spec' | 'none',
```